Import new splash screen and systray-icon

zander opened the pull request:
Artwork by Mick Bruce mickbruce.mjb@gmail.com

zander committed:
Import new splash screen and systray-icon

Artwork by Mick Bruce <mickbruce.mjb@gmail.com>

dskloet commented:
Why does (so much) code need to be changed just to change an image?

christophebiocca commented:
Tested and it works (ArchLinux; XFCE4).

~~@dskloet: My guess is that he modified the 0.12 version of the file, and is now backporting those code changes inadvertently?~~

zander commented:
I didn't touch the 0.12 branch.

The reason for the amount of code is based on the differences in content and some bad code in the old version.

We now have a individual images for splash and (systray) icon. Both needed to be colourized individually, hence some refactor.
At the same time the usage of wrong classes (QICon and QPixmap) was removed to avoid lots of copying and converting of data. The main effect of that was that I could now use a proper painting method so resizing the splash to the target display DPI looks good.

rromanchuk commented:
Tested ACK (with 0.12 branch)

rromanchuk commented:
@zander just a pet peeve of mine, while you're in this area, anyway you can change the tooltip from bitcoin-qt (this is confusing engineer speak to the end user) to Bitcoin Classic, the name of the application they are actually running.

jtoomim commented:
utACK.

jgarzik commented:
Looks good from a quick scan.  Will test this weekend, hopefully merge this weekend.

Nice work!

zander committed:
New copyright year

zander commented:
Rebased and added new copyright year.

People asked what it looks like;
![splash](https://cloud.githubusercontent.com/assets/63063/12723499/a1634784-c901-11e5-8d7a-75d2f4d90f37.png)
![about](https://cloud.githubusercontent.com/assets/63063/12723501/a38bc4fa-c901-11e5-9fe6-edad18acc130.png)

The old code still adjusts for the end-users aspect ratio based on DPI (mine is 96x96).

pdpi commented:
LGTM.

zander commented:
Rebased to fix merge conflict in configure.ac

sandakersmann commented:
utACK

zander commented:
I'll revisit for a future branch.

hikari-no-yume commented on src/clientversion.h:
```diff
@@ -38,7 +38,7 @@
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Developers"
```
This kinda strips both Core and Classic of attribution. Perhaps you should attribute both of them? Leave the Core message intact, and add another for Classic.
